### PR TITLE
Added note in sections of Start times and Start times for a day

### PR DESCRIPTION
**Pull request by yukariemma**
@otty

### CODE REVIEW

File: source/includes/v4/_tours.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ curl "https://demo.gomus.de/api/v4/tours/categories"
 
 ### Response
 
-The json response contains a list of all tour categories to build up filters. 
+The json response contains a list of all tour categories to build up filters.
```
Comment (huseyin) [inline source/includes/v4/_tours.md:32]:
```suggestion
The JSON response contains a list of all tour categories to build up filters.
```

File: source/includes/v4/_tours.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ curl "https://demo.gomus.de/api/v4/tours/languages"
 
 ### Response
 
-The json response contains a list of all languages used by online bookable tour products to build up filters. 
+The json response contains a list of all languages used by online bookable tour products to build up filters.
```
Comment (huseyin) [inline source/includes/v4/_tours.md:63]:
```suggestion
The JSON response contains a list of all languages used by online bookable tour products to build up filters.
```

**Review by huseyin — CHANGES_REQUESTED** [review]

File: source/includes/v4/_tours.md (hunk 1 above)
Comment (douglasward) [reply source/includes/v4/_tours.md:32]: Thank you very much for point this out, @huseyin. You are of course very correct - json should be JSON. I have created a new pull request which corrects all of the occurences in the repository where this is the case. See: https://github.com/giantmonkey/gomus-api-doc/pull/30

File: source/includes/v4/_tours.md (hunk 2 above)
Comment (douglasward) [reply source/includes/v4/_tours.md:63]: see: https://github.com/giantmonkey/gomus-api-doc/pull/30

**Pull request merged**